src/features/CourtCaseList/CourtCaseListEntry.tsx

```diff
@@ -0,0 +1,184 @@
+import { deleteQueryParamsByName } from "utils/deleteQueryParam"
+import getTriggerWithDescription from "utils/formatReasons/getTriggerWithDescription"
+import groupErrorsFromReport from "utils/formatReasons/groupErrorsFromReport"
+import { displayedDateFormat } from "utils/formattedDate"
+import {
+  filterUserNotes,
+  first100CharsOfMostRecentNote,
+  getMostRecentNote,
+  validatedMostRecentNoteDate
+} from "./CourtCaseListEntryHelperFunction"
+import { NotePreview, NotePreviewButton } from "./NotePreviewButton"
+import LockedByTag from "./tags/LockedByTag"
+import ResolvedTag from "./tags/ResolvedTag"
+import UrgentTag from "./tags/UrgentTag"
+
+const useStyles = createUseStyles({
+  caseDetailsRow: {
+    verticalAlign: "top",
+    borderColor: "unset"
+  },
+  triggersRow: {
+    verticalAlign: "top",
+    backgroundColor: "#f3f2f1" // GDS light-grey color
+  },
+  flexBox: {
+    display: "flex"
+  },
+  notesRow: {
+    borderTop: "white solid"
+  }
+})
+
+interface Props {
+  courtCase: CourtCase
+  currentUser: User
+}
+
+const CourtCaseListEntry: React.FC<Props> = ({ courtCase, currentUser }: Props) => {
+  const {
+    errorId,
+    courtDate,
+    ptiurn,
+    defendantName,
+    courtName,
+    triggers,
+    errorReport,
+    isUrgent,
+    errorLockedByUsername,
+    triggerLockedByUsername,
+    resolutionTimestamp,
+    notes
+  } = courtCase
+  const classes = useStyles()
+  const { basePath, query } = useRouter()
+  let searchParams = new URLSearchParams(encode(query))
+  searchParams = deleteQueryParamsByName(["unlockException", "unlockTrigger"], searchParams)
+  const caseDetailsPath = (id: number) => `${basePath}/court-cases/${id}`
+  const unlockCaseWithReasonPath = (reason: "Trigger" | "Exception", caseId: string) => {
+    searchParams.append(`unlock${reason}`, caseId)
+    return `${basePath}/?${searchParams}`
+  }
+  const canUnlockCase = (lockedUsername: string): boolean => {
+    return currentUser.groups.includes("Supervisor") || currentUser.username === lockedUsername
+  }
+
+  const exceptions = groupErrorsFromReport(errorReport)
+  const [showPreview, setShowPreview] = useState(false)
+  const userNotes = filterUserNotes(notes)
+  const numberOfNotes = userNotes.length
+  const mostRecentUserNote = getMostRecentNote(userNotes)
+
+  const triggerRow =
+    triggers.length > 0 ? (
+      <Table.Row className={classes.triggersRow}>
+        <Table.Cell>
+          <ConditionalRender isRendered={!!triggerLockedByUsername}>
+            <Image src={"/bichard/assets/images/lock.svg"} width={20} height={20} alt="Lock icon" />
+          </ConditionalRender>
+        </Table.Cell>
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell>
+          {triggers?.map((trigger, triggerId) => (
+            <GridRow key={`trigger_${triggerId}`}>{getTriggerWithDescription(trigger.triggerCode)}</GridRow>
```

Comment:
Like above with `index`.

Reply:
This was not changed as part of this ticket and is old code.